kikit/stencil.py

```diff
@@ -311,8 +312,9 @@ def create(inputboard, outputdir, jigsize, jigthickness, pcbthickness,
         ("PasteBottom", pcbnew.B_Paste, "Paste Bottom"),
         ("PasteTop", pcbnew.F_Paste, "Paste top"),
     ]
+    exportSettings["ExcludeEdgeLayer"] = True
```

Comment:
This command modifies the global variable. That means that any other reference to "exportSettings" will have a modified value. It makes no difference here as the program immediately exits when invoked from CLI, but if a user writes a script, that might make a difference (and pretty hard to track). Please, make a copy of the dictionary first and then change the value.

Reply:
That is an absolutely valid point. 
I have now imported the dictionary with the original name to have an easy way to change it. A copy is then created with the name "exportSettings".

Maybe it would be an even better option to create a settings dictionary (e.g. "exportSettingsStencil") directly in export.py and import that dict into stencil.py to avoid a coupling with settings defined for a specific fab.

Reply:
The decoupling is currently on a roadmap; it will probably happen soon after CLI rework. Therefore, I am happy with this PR as is.